src/solvers.jl

```diff
@@ -111,6 +110,15 @@ function solve(m::Model; suppress_warnings=false,
                 fill(NaN, numRows)
             end
         end
+        # conic duals
+        if traits.conic
+            m.conicconstrDuals = try
+                MathProgBase.getconstrduals(m.internalModel)
```

Comment:
There is a function under quad_to_conic.jl that calls getconicdual inside, thought I should have called that, switching asap!